third_party/docfx-doclet-143274/src/main/java/com/microsoft/lookup/BaseLookup.java

```diff
@@ -313,4 +327,110 @@ private String getSeeTagRef(SeeTree seeTree) {
     return String.format(
         "<xref uid=\"%1$s\" data-throw-if-not-resolved=\"false\">%1$s</xref>", ref);
   }
+
+  public String extractStatus(T element) {
+    List<String> annotationNames =
+        element.getAnnotationMirrors().stream()
+            .map(mirror -> mirror.getAnnotationType().asElement().getSimpleName().toString())
+            .collect(Collectors.toList());
+
+    if (annotationNames.stream().anyMatch("InternalApi"::equals)
+        || annotationNames.stream().anyMatch("InternalExtensionOnly"::equals)) {

```

Comment:
Do you think it would make sense to separate them? I believe there are slight differences between the two: https://github.com/googleapis/sdk-platform-java/tree/main/gax-java#versioning

From my understanding (could be wrong), I believe `@InternalExtensionOnly` is okay for certain use cases, but `@Internal` is never.

Reply:
I added more of an explanation below to Alice's similar comments. It might make sense to add better clarifying descriptions since two people had the same question.

Reply:
Er, to reframe the question: Why do `@Internal` and `@InternalExtensionOnly` both return `internal` and not `internal` and `internalextensiononly` respectively?

I believe `InternalExtensionOnly` is not truly internal, but just a more restrictive public interface/class (in regards to stability).

Reply:
Ah. I see - sorry for misunderstanding. You're right this is inconsistent with the rest of the PR - I did this earlier than the rest and didn't revisit it. It turns out these statuses don't do anything except for "deprecated" and "beta", so I'm going to remove the others.